base-permissions/src/commonMain/kotlin/PermissionManager.kt

```diff
@@ -61,64 +61,32 @@ interface PermissionManager<P : Permission> {
 abstract class BasePermissionManager<P : Permission>(
     override val permission: P,
     private val settings: Settings,
-    private val coroutineScope: CoroutineScope
+    protected val coroutineScope: CoroutineScope
```

Comment:
This class already uses delegation and `this` can be used as `CoroutineScope` when needed, so maybe should't be a property at all?